pact/fixed-quote-policy.pact

```diff
@@ -79,7 +79,11 @@
     )
     @doc "Capture quote spec for SALE of TOKEN from message"
     (enforce-sale-pact sale-id)
-    (let ( (spec:object{quote-spec} (read-msg QUOTE) ) )
+    (let ( (spec:object{quote-spec} (read-msg QUOTE) )
+           (fungible:module{fungible-v2} (at 'fungible (read-msg QUOTE)) )
```

Comment:
```suggestion
           (fungible:module{fungible-v2} (at 'fungible spec) )
```

Reply:
This doesn't work - `Cannot resolve "spec"` unless we make nested let block

Reply:
yes, please use `let*`

Reply:
Oh, fixed.